### PR TITLE
MsTextarea

### PR DESCRIPTION
**Pull request by Ironicbay**

### CODE REVIEW

File: lib/components/ms-textarea/MsTextarea.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    >
+      <ion-icon
+        class="form-error-icon"
+        :icon="warning"
+      />
+      {{ $msTranslate(errorMessage) }}
+    </span>
+    <span
+      class="count"
+      v-show="showCount || true"
```
Comment (Max-7) [inline lib/components/ms-textarea/MsTextarea.vue:41]: `|| true`?

File: lib/components/ms-textarea/MsTextarea.vue [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    >
+      {{ modelValue?.length }} / {{ maxlength || DEFAULT_MAXLENGTH }}
```
Comment (Max-7) [inline lib/components/ms-textarea/MsTextarea.vue:43]: What happens if you set `:showCount="true"` without `v-model`?

File: lib/components/ms-textarea/MsTextarea.vue [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  disabled?: boolean;
+  autogrow?: boolean;
+  rows?: number;
+  minlength?: number;
+  maxlength?: number;
+  showCount?: boolean;
+}>();
+
+const textareaRef = ref();
+const errorMessage: Ref<Translatable> = ref('');
```
Comment (Max-7) [inline lib/components/ms-textarea/MsTextarea.vue:74]: Seems useless.

File: lib/components/ms-textarea/MsTextarea.vue [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        input: !$props.disabled,
+      }"
+      ref="textareaRef"
+      :placeholder="$msTranslate(placeholder)"
+      :value="modelValue"
+      @ion-input="onChange($event.detail.value || '')"
+      :disabled="$props.disabled"
+      v-bind="$attrs"
+      :auto-grow="autogrow || false"
+      :minlength="minlength || DEFAULT_MINLENGTH"
```
Comment (Max-7) [inline lib/components/ms-textarea/MsTextarea.vue:25]: Is there a reason we want to set default values for these props? It seems easier to get rid of them, just `v-bind="$attrs"` and let ionic handle the behavior.

**Review by Max-7 — CHANGES_REQUESTED** [review]

File: lib/components/ms-textarea/MsTextarea.vue (hunk 1 above)
Comment (Ironicbay) [reply lib/components/ms-textarea/MsTextarea.vue:41]: Showing the count by default. I guess it doesn't matter if it's disabled by default

File: lib/components/ms-textarea/MsTextarea.vue (hunk 2 above)
Comment (Ironicbay) [reply lib/components/ms-textarea/MsTextarea.vue:43]: `/ maxlength` displays.

**Review by Max-7 — APPROVED** [review]

**Pull request merged**